### DISCUSSION TITLE
Are there Include Lagrange multipliers in results from AC- OPF or DC OPF?

### DISCUSSION THREAD

**Discussion by Crueda96** [bukhsh/oats#127] [Q&A]
Is it possible to calculate the nodal LMP prices for each bus in the AC OPF or DC OPF?

**Comment by bukhsh** [comment]
It is possible to include that information in the output, and the command to retrieve this information depends on the choice of the solver.

What solver are you using?

**Reply by Crueda96** [reply]
I am using the glpk or gurobi solver
